NSString+MD5.h

```diff
@@ -7,8 +7,25 @@
 
 @import Foundation;
 
+
+#ifdef NS_ASSUME_NONNULL_BEGIN
+#define MD5_DIGEST_ASSUME_NONNULL_BEGIN NS_ASSUME_NONNULL_BEGIN
```

Comment:
Yeah, that’s a good idea. I’ve just been doing this for a few internal libraries and it became a reflex!